src/core/Form/TextInput/TextInput.md

```diff
@@ -13,18 +13,19 @@ import { TextInput } from 'suomifi-ui-components';
 <>
   <TextInput
     onBlur={event => console.log(event.target.value)}
-    labelMode="screenreader"
+    labelMode="hidden"
     labelText="Test with hidden label"
+    defaultValue="Test with hidden label"
   />
   <TextInput.error
-    labelMode="screenreader"
+    labelMode="hidden"
     labelText="Error with hidden label"
-    value="Error with hidden label"
+    defaultValue="Error with hidden label"
```

Comment:
Replaced value with defaultValue to make the example inputs interactive in the styleguide. Text couldn't be edited when label was set the way it was.